packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -178,6 +206,24 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
       $vertical-padding: var(--onyx-input-padding-vertical)
     );
 
+    &__header {
+      display: flex;
+      max-width: 100%;
+      width: 100%;
+    }
+
+    &__info-message {
+      height: 1rem;
+      align-self: center;
+    }
+
+    &__tooltip-trigger {
```

Comment:
Should we create a small support component to do this? Since we need it two times here and also in the textarea and the select? Maybe something like "OnyxInfoTooltip"?

Reply:
I agree, a support component would be helpful. But should we communicate this with Linda and Jonathan first?

Reply:
I'd say you can already implement it and we can show it to Linda in the daily on Monday